GitCommands/Git/GitModule.cs

```diff
@@ -1826,7 +1826,7 @@ private static void UpdateIndex(StreamWriter inputWriter, string? filename)
         {
             var bytes = EncodingHelper.ConvertTo(
                 SystemEncoding,
-                $"{inputWriter.NewLine}\"{filename.ToPosixPath()}\"");
+                $"\"{filename.ToPosixPath()}\"{inputWriter.NewLine}");
```

Comment:
Why has this changed?

Reply:
> Fixup GitModule.UpdateIndex: output the newline after the file name
in order to avoid a git error message on the first file name being empty